### PR TITLE
React validated input component

### PR DESCRIPTION
**Pull request by stefancarrera**
react-validation

### CODE REVIEW

File: react-validated-input-component/dist/styles.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
+.fa-times {
+  margin-left: 0.5rem;
+  color: red;
+}
+
+.fa-check {
+  margin-left: 0.5rem;
+  color: green;
+}
```
Comment (scbowler) [inline react-validated-input-component/dist/styles.css:9]: You don't want to add css rules to existing classes from a css library. This changes the default behavior and cause issues in other parts of your code and cause confusion with other people working on the project. Here you can just make a couple custom classes and use those in conjunction with the classes from the css library. For example:

```CSS
.icon {
  margin-left: 0.5rem;
}

.icon-red {
  color: red;
}

.icon-green {
  color: green;
}
```

File: react-validated-input-component/src/validated-input.jsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    this.setState({ value: event.target.value });
+  }
+
+  render() {
+    const password = this.state.value;
+    let p = '';
+    let icon = '';
+    if ((password.length <= 7) && (password.length >= 1)) {
+      p = <p>Your password is too short</p>;
+      icon = <i className="fas fa-times"></i>;
```
Comment (scbowler) [inline react-validated-input-component/src/validated-input.jsx:23]: Here you can utilize your classes:

```JSX
icon = <i className="fas fa-times icon icon-red"></i>;
```

And then the same for the other icons below

**Review by scbowler — APPROVED** [review]
Looks good! 👍

**Review by scbowler — CHANGES_REQUESTED** [review]
Did not mean to approve, a few changes needed, sorry about that

**Comment by stefancarrera** [comment]
Fixed the CSS double rules

**Review by scbowler — APPROVED** [review]
Good job implementing the changes, looks great! 👍

**Pull request merged**